instrument_api/src/test/java/com/example/instrument_api/InstrumentControllerIntegrationTests.java

```diff
@@ -1,4 +1,199 @@
+                .andExpect(jsonPath("$[1].musicians[0].id", is(4)))
+                .andExpect(jsonPath("$[1].musicians[0].name", is("Bartolo Alvarez")))
+                .andExpect(jsonPath("$[1].musicians[0].yearOfBirth", is(1914)))
+                .andExpect(jsonPath("$[1].musicians[0].yearOfDeath", is(2017)))
+
+                .andExpect(jsonPath("$[2].id", is(5)))
+                .andExpect(jsonPath("$[2].name", is("Trumpet")))
+                .andExpect(jsonPath("$[2].description", is("If you think the brass family got its name because the instruments are made of brass, you're right! This family of instruments can play louder than any other in the orchestra and can also be heard from far away. Although their early ancestors are known to have been made of wood, tusks, animal horns or shells, today's modern instruments are made entirely of brass. Brass instruments are essentially very long pipes that widen at their ends into a bell-like shape. The pipes have been curved and twisted into different shapes to make them easier to hold and play.")))
+                .andExpect(jsonPath("$[2].period", is("1982")))
+                .andExpect(jsonPath("$[2].collection", is("Brass")))
+                .andExpect(jsonPath("$[2].musicians", hasSize(1)))
+                .andExpect(jsonPath("$[2].musicians[0].id", is(6)))
+                .andExpect(jsonPath("$[2].musicians[0].name", is("Irving Fields")))
+                .andExpect(jsonPath("$[2].musicians[0].yearOfBirth", is(1915)))
+                .andExpect(jsonPath("$[2].musicians[0].yearOfDeath", is(2016)))
+
+                .andExpect(jsonPath("$[3].id", is(7)))
+                .andExpect(jsonPath("$[3].name", is("Piano")))
+                .andExpect(jsonPath("$[3].description", is("This class of musical instruments requires you to blow into a specific wind instrument by following an order to ensure that the sound that you desire is produced. The instruments can be expected to work depending on the principles of frequencies, sound waves, acoustics, resonance and harmonics. The pitch of the produced sound when you start blowing the instrument is actually dependent on the length of the air column through which the waves of the sounds vibrate")))
+                .andExpect(jsonPath("$[3].period", is("1980")))
+                .andExpect(jsonPath("$[3].collection", is("Percussion")))
+                .andExpect(jsonPath("$[3].musicians", hasSize(1)))
+                .andExpect(jsonPath("$[3].musicians[0].id", is(8)))
+                .andExpect(jsonPath("$[3].musicians[0].name", is("Ray Anthony")))
+                .andExpect(jsonPath("$[3].musicians[0].yearOfBirth", is(1922)))
+                .andExpect(jsonPath("$[3].musicians[0].yearOfDeath", is(2022)))
+
+        ;
+
+    }
+    // GET /instrument/name/{name}: Instrument
+    @Test
+    public void givenInstrument_whenGetInstrumentByName_thenReturnJsonReview() throws Exception {
```

Comment:
ook weer, je geeft geen instrument, en returnt geen reviews. Beter: whenGetInstrumentByName_thenReturnInstrumentAndMusicians of zoiets